google-cloud-pub-sub/src/test/scala/akka/stream/alpakka/googlecloud/pubsub/impl/PubSubApiSpec.scala

```diff
@@ -244,17 +250,17 @@ class PubSubApiSpec extends AnyFlatSpec with BeforeAndAfterAll with ScalaFutures
       WireMock
         .post(
           urlEqualTo(
-            s"/v1/projects/${config.projectId}/subscriptions/sub1:pull"
+            s"/v1/projects/${TestCredentials.projectId}/subscriptions/sub1:pull?prettyPrint=false"
```

Comment:
How come prettyPrint is disabled?

Reply:
`prettyPrint=false` is the default because otherwise (at least some) Google APIs return huge JSON responses full of whitespace. We could set `prettyPrint=true` for tests?

Reply:
It might make debugging easier in the future.

Reply:
I'm going to leave this for future PRs to decide on a per-module basis. In my case setting `prettyPrint=false` for BigQuery actually made debugging easier, albeit due to the terrible formatting of the Hoverfly recording/replay scenario:
https://github.com/akka/alpakka/blob/master/google-cloud-bigquery/src/test/resources/BigQueryEndToEndSpec.json#L81